### DISCUSSION TITLE
Get nitro credit

### DISCUSSION THREAD

**Discussion by drewold123** [Vedza/NitroSniperGo#105] [Ideas]
If you already have nitro can you make it that it adds it as a nitro credit. 

I'm currently using your python nitro sniper and i got nitro from it but i think i could have got more (in credits) but it doesnt claim it since i have nitro.

**Comment by Vedza** [comment]
It does add it to credit, but discord might disable it sometimes if you try to claim too much Nitro

**Reply by drewold123** [reply]
is that the same with your python sniper

**Reply by Vedza** [reply]
Yes but you should definitely use this one instead

**Reply by drewold123** [reply]
i don't think the the python sniper adds to credit but i will try the go lang sniper i just have never used go

**Reply by Vedza** [reply]
You can just download binaries in release. And about the redeeming part, it works the same as the python one and that's discord that automatically put it in credit if you already have Nitro
